spec/support/active_admin_helpers.rb

```diff
@@ -0,0 +1,58 @@
+module ActiveAdminHelpers
+  extend self
+
+  def reload_menus!
+    ActiveAdmin.application.namespaces.each{|n| n.reset_menu! }
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Pass `&:reset_menu!` as an argument to `each` instead of a block.